tv/tv.cpp

```diff
@@ -226,7 +227,7 @@ struct TVLegacyPass final : public llvm::ModulePass {
     bool regenerate_tgt = verify(t, I->second.n++, I->second.fn_tostr);
 
     if (regenerate_tgt) {
-      I->second.fn = *llvm2alive(F, *TLI);
+      I->second.fn = *llvm2alive(F, *TLI, false);
```

Comment:
> The easiest solution is to just drop regenerate_tgt and assume it's always true (just forget about the optimization we have). Otherwise, we would need to track whether some metadata was dropped.

Thanks I went with that option.

> Another option is to bug me about the metadata you need me to implement 😅

Yeah, I guess it still makes sense to have the option to ignore it, as it seems unfortunate that some unknown metadata prevents verification of test cases.